button-examples/Web/web-button-example.md

```diff
@@ -133,13 +138,29 @@ module.exports = {
 };
 ```
 
+Import base styles of button into your `*.scss` stylesheet using :
+
+<details><summary><b<Expand for theming instructions for <code>mdc-button</code> or <code>mdc-icon-button</code>.</b></summary>
+
+**mdc-button**
+```scss
+@use '@material/button/mdc-button';
```

Comment:
This is a new Sass syntax which'll be available in MDC Web `v5.0`. Going forward all the usage documentation will follow this syntax.